src/Css/Style.php

```diff
@@ -1551,6 +1559,23 @@ protected function computed(string $prop)
             $this->_props_computed[$prop] = $computed;
         }
 
+        // now compute value of css var access (late binding)
+        if(isset($this->_props_computed[$prop])) {
+            $value = $this->_props_computed[$prop];
+            if(!is_array($value)) {
+		    $value = [$value];
+            }
+            foreach($value as &$part) {
+                if(is_string($part) && substr($part, 0, 4) == 'var(') {
+                    $css_var = str_replace("-", "_", substr(trim($part), 4, -1));
+                    $part = $this->get_specified($css_var) ?? (isset($this->parent_style) ? $this->parent_style->get_specified($css_var) : null);
+		}
+            }
+            if(!is_array($this->_props_computed[$prop])) {
+                $this->_props_computed[$prop] = $value[0];
+            }
+        }
+
```

Comment:
This should probably be handled in the compute_prop method so that the value can then be parsed by any property-specific compute methods.

Reply:
Is this logic sufficient to accommodate something like:

```
<style>
    :root {
        --half: 50%;
    }
    div {
        width: var(--half);
    }
</style>
<div>dompdf</div>
```